### PR TITLE
[FSSDK-9493] Fix: FetchQualifiedSegments without options, crash fixed

### PR DESCRIPTION
**Pull request by muzahidul-opti**
## Summary
- Fix the crush issue for android `fetchQualifiedSegments` API when no options provided.

## Test plan
-  All previous tests should pass

## Issues
- FSSDK-9493

### CODE REVIEW

**Review by yasirfolio3 — APPROVED** [review]
LGTM

**Review by jaeopt — CHANGES_REQUESTED** [review]
Can we have a test case to cover the crash?

File: android/src/main/java/com/optimizely/optimizely_flutter_sdk/helper_classes/Utils.java [hunk 1]
```diff
@@ -72,7 +72,7 @@ public static List<OptimizelyDecideOption> getDecideOptions(List<String> options
     }
 
     public static List<ODPSegmentOption> getSegmentOptions(List<String> options) {
-        if(options == null || options.isEmpty()) {
+        if(options == null) {
```
[the remark this answers is not in the document]
Comment (yasirfolio3) [reply android/src/main/java/com/optimizely/optimizely_flutter_sdk/helper_classes/Utils.java:75]: Flutter sends empty options list to native sdk's in case no options are provided. The removed check causes the options to be null for empty list (which is default flutter behaviour) which is not supported by native android-sdk.

[the remark this answers is not in the document]
Comment (yasirfolio3) [reply android/src/main/java/com/optimizely/optimizely_flutter_sdk/helper_classes/Utils.java:75]: @muzahidul-opti can we add a unit test that verifies that in case no options are provided, flutter sends empty list to native sdk's? and then add an integration test to @optimizely-flutter-testapp to validate the response.
